src/main/de/linearbits/preferences/EditorBoolean.java

```diff
@@ -83,7 +95,11 @@ Boolean parse(final String s) {
     void setValue(Object t) {
         setInitialValue((Boolean) t);
         checkbox.setSelection((Boolean) t);
-        checkbox.setText((Boolean) t ? getDialog().getConfiguration().getStringYes() : getDialog().getConfiguration().getStringNo());
+        if (! hideLabel){
```

Comment:
Redundant blank before "hideLabel". Blank is missing before "{" -> Careful editing / autoformat.